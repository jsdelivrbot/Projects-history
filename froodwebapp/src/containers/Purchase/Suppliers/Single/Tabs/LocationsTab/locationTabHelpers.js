import React from 'react';
import { TableActionButtons } from 'components';

export default (handleEdit, handleDeactivate) => [{
  title: 'Name',
  dataIndex: 'name',
}, {
  title: 'Address',
  dataIndex: 'address1',
  width: '30%'
}, {
  title: 'City',
  dataIndex: 'cityName',
}, {
  title: 'State',
  dataIndex: 'stateName',
}, {
  title: 'Postal Code',
  dataIndex: 'postalCode',
}, {
  title: 'Action',
  render: (text, record) => (
    <TableActionButtons
      record={ record }
      handleEdit={ handleEdit }
      handleDeactivate={ handleDeactivate }
      deleteButtonVisible
    />
  )
}];
