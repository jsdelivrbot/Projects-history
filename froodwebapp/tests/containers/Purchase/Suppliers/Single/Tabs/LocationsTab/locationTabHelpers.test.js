import locationTabHelpers from 'containers/Purchase/Suppliers/Single/Tabs/LocationsTab/locationTabHelpers';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const { describe, expect, it } = global;

describe('locationTabHelpers', () => {
  const props = {
    handleActivate: jest.fn(),
    handleDeactivate: jest.fn()
  };

  it('renders the locationTabHelpers component', () => {
    const component = shallow(<locationTabHelpers { ...props } />);

    expect(component.length).toEqual(1);
  });

  it('capturing Snapshot of locationTabHelpers', () => {
    const component = shallow(<locationTabHelpers { ...props } />);
    const view = toJson(component);
    const actual = expect(view);

    actual.toMatchSnapshot();
  });
});
