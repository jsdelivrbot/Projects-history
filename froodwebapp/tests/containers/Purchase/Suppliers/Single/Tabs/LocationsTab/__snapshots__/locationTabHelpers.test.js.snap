// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`locationTabHelpers capturing Snapshot of locationTabHelpers 1`] = `
<locationTabHelpers
  handleActivate={[Function]}
  handleDeactivate={[Function]}
/>
`;
